experimenter/experimenter/experiments/api/v5/serializers.py

```diff
@@ -1531,6 +1536,79 @@ def _validate_bucket_duplicates(self, data):
 
         return data
 
+    def _validate_localization_content(self, data):
```

Comment:
We also need a check for application is desktop here since this only applies to desktop.